scripts/install/opt/seagate/cortx/motr/bin/motr_mini_prov.py

```diff
@@ -428,6 +425,33 @@ def get_nodes(self):
         nodes.append(value["hostname"])
     return nodes
 
+def get_data_disks_count(self):
+    nodes_info = Conf.get(self._index, 'server_node')
+    total_disks = 0
+    for node in nodes_info.values():
+        if 'storage' in node:
+            storage = node['storage']
+            cvg_count = storage['cvg_count']
+            for i in range(int(cvg_count)):
+                total_disks += len(storage["cvg"][i]["data_devices"])
+    return total_disks
+
+def check_data_disks_count(self):
+    total_disks = get_data_disks_count(self)
+    required_disks = get_data_parity_spare_count(self)
+    if (total_disks <= required_disks):
```

Comment:
if variable name is required_disks, then total_disks <= required_disks sounds like a proper condition but we are rasising failure on this condition.